shared/checks/oval/installed_OS_is_wrlinux8.xml

```diff
@@ -1,37 +1,37 @@
 <def-group>
-  <definition class="inventory" id="installed_OS_is_wrlinux" version="1">
+  <definition class="inventory" id="installed_OS_is_wrlinux8" version="1">
     <metadata>
-      <title>WRLinux</title>
+      <title>WRLinux 8</title>
       <affected family="unix">
         <platform>multi_platform_all</platform>
       </affected>
       <reference ref_id="cpe:/o:windriver:wrlinux"
       source="CPE" />
       <description>The operating system installed on the system is
-      Wind River Linux</description>
+      Wind River Linux 8</description>
     </metadata>
     <criteria>
-      <criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_wrlinux" />
-      <criterion comment="WRLinux is installed" test_ref="test_wrlinux" />
+      <criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_wrlinux8" />
+      <criterion comment="WRLinux 8 is installed" test_ref="test_wrlinux8" />
     </criteria>
   </definition>
 
-  <ind:family_test check="all" check_existence="at_least_one_exists" comment="installed OS part of unix family" id="test_unix_wrlinux" version="1">
-    <ind:object object_ref="obj_unix_wrlinux" />
-    <ind:state state_ref="state_unix_wrlinux" />
+  <ind:family_test check="all" check_existence="at_least_one_exists" comment="installed OS part of unix family" id="test_unix_wrlinux8" version="1">
+    <ind:object object_ref="obj_unix_wrlinux8" />
+    <ind:state state_ref="state_unix_wrlinux8" />
   </ind:family_test>
 
-  <ind:family_state id="state_unix_wrlinux" version="1">
+  <ind:family_state id="state_unix_wrlinux8" version="1">
     <ind:family>unix</ind:family>
   </ind:family_state>
 
-  <ind:family_object id="obj_unix_wrlinux" version="1" />
+  <ind:family_object id="obj_unix_wrlinux8" version="1" />
 
-  <file_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" check_existence="at_least_one_exists" version="1" id="test_wrlinux" check="all" comment="Test presence of /etc/wrlinux-release.">
-      <object object_ref="obj_test_wrlinux" />
+  <file_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" check_existence="at_least_one_exists" version="1" id="test_wrlinux8" check="all" comment="Test presence of /etc/wrlinux-release.">
+      <object object_ref="obj_test_wrlinux8" />
   </file_test>
 
-  <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="obj_test_wrlinux" >
+  <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="obj_test_wrlinux8" >
```

Comment:
Is the file `/etc/wrlinux-release` exclusive to WRLinux8?
I recommend to make the check more specific for version 8.

Reply:
Thanks for the remind.
WRLinux8 use /etc/wrlinux-release to record the relevant version message but WRLinux1019 does not use it any more. I will check more version message in /etc/wrlinux-release file for WRLinux8, and submit another version pull request